src/l500/l500-device.cpp

```diff
@@ -435,12 +435,26 @@ namespace librealsense
     {
         // Stop all data streaming/exchange pipes with HW
         stop_activity();
+        using namespace std;
+        using namespace std::chrono;
```

Comment:
In this case use of explicit  std::this_thread... `std::chrono:millisec(..)` is preferred